WordPress/src/main/java/org/wordpress/android/ui/stats/refresh/lists/detail/PostYearsMapper.kt

```diff
@@ -1,18 +1,25 @@
 package org.wordpress.android.ui.stats.refresh.lists.detail
 
+import org.wordpress.android.fluxc.model.stats.PostDetailStatsModel
 import org.wordpress.android.fluxc.model.stats.PostDetailStatsModel.Year
+import org.wordpress.android.fluxc.network.utils.StatsGranularity.DAYS
 import org.wordpress.android.ui.stats.refresh.lists.sections.BlockListItem
 import org.wordpress.android.ui.stats.refresh.lists.sections.BlockListItem.Divider
 import org.wordpress.android.ui.stats.refresh.lists.sections.BlockListItem.ExpandableItem
 import org.wordpress.android.ui.stats.refresh.lists.sections.BlockListItem.ListItemWithIcon
 import org.wordpress.android.ui.stats.refresh.lists.sections.BlockListItem.ListItemWithIcon.TextStyle.LIGHT
+import org.wordpress.android.ui.stats.refresh.utils.StatsDateFormatter
 import org.wordpress.android.ui.stats.refresh.utils.toFormattedString
 import org.wordpress.android.util.LocaleManagerWrapper
 import java.text.DateFormatSymbols
+import java.util.Date
 import javax.inject.Inject
 
 class PostYearsMapper
```

Comment:
Now that it's used for other types of granularity, perhaps the name should be updated. What do you think?